securedrop/journalist.py

```diff
@@ -292,6 +296,80 @@ def admin_delete_user(user_id):
     return redirect(url_for('admin_index'))
 
 
+@app.route('/account', methods=('GET', 'POST'))
+@login_required
+def edit_account():
+    user = g.user
+
+    if request.method == 'POST':
+        if request.form['password'] != "":
+            if request.form['password'] != request.form['password_again']:
+                flash("Passwords didn't match", "error")
+                return redirect(url_for("edit_account"))
+            try:
+                user.set_password(request.form['password'])
+            except InvalidPasswordLength:
+                flash("Your password is too long "
+                      "(maximum length {} characters)".format(
+                      Journalist.MAX_PASSWORD_LEN), "error")
+                return redirect(url_for("edit_account"))
+
+        try:
+            db_session.add(user)
+            db_session.commit()
+            flash(
+                "Password successfully changed!",
+                "notification")
+        except Exception as e:
+            db_session.rollback()
+            flash(
+                "An unknown error occurred, please inform your administrator",
```

Comment:
In order for the administrator to be able to respond effectively, they need some info about what happened. For now, can you add a call to `app.logger.error` with the contents of `e` so they'll at least have something to go off of?

In general, this exception handling needs an overhaul but at least logging the error is a start that will aid future debugging and incident response.